bottles/backend/wine/winecommand.py

```diff
@@ -517,11 +517,9 @@ def get_cmd(
 
                 # Create temporary sh script in /tmp where Gamescope will execute it
                 file = [f"#!/usr/bin/env sh\n"]
+                file.append(f"{command} $@")
```

Comment:
The `"$@"` Special Parameter should usually be written within double-quotes, otherwise it'll be subjected to Field Splitting and Pathname Expansion (aka globbing). Is either of these behaviors desired here?